api/spec/v1.yaml

```
@@ -767,17 +767,17 @@ components:
       properties:
         latest_chain_id:
           type: string
-          description: The latest chain ID at the head of indexing.
+          description: The most recently indexed chain ID.
           example: *chain_id_1
         latest_block:
           type: integer
           format: int64
-          description: The latest indexed block at the head of indexing.
+          description: The most recent indexed block. Query a synced Oasis node for the latest block produced.
           example: *block_height_1
         latest_update:
           type: string
           format: date-time
-          description: The RFC 3339 formatted time of latest indexing update.
+          description: The RFC 3339 formatted time of the most recently indexed block. Check for drift with a synced Oasis node.
```

Nit: I wouldn't recommend comparing with a live node here. If you compare this timestamp with the timestamp of the latest block, you're getting the exact same info as from `latest_block`. I see the utility of this field more so in the fact that the value is (most of the time) meaningful without node access.

Possible wording:

```suggestion
          description: The RFC 3339 formatted time of the most recently indexed block. To see how much the index lags behind the live blockchain, compare this timestamp to the current time for a rough estimate, or use `latest_block` do determine the lag exactly.
```